foundation_pose_sam3/NamedPoseStamped[] poses